### PR TITLE
Add `_set_fitted_parameters` method

### LINKED ISSUE #828: Add `_set_fitted_parameters` method to `UnixTimestampEncoder`
### Problem Description
We'd like to be able to get a transformer into a fitted state using known parameters instead of existing data. This would allow us to set up a transformer to be able to reverse transform without first having to fit it on real data.

### Expected behavior
Add the method `_set_fitted_parameters` to the `UnixTimestampEncoder`. When called, the provided arguments should be set on the transformer to get it into a 'fitted' state so that it can be used to reverse transform. After being called, a user should be able to call `reverse_transform` and have it work as expected.

def _set_fitted_parameters(self, column_names, min_max_values, null_transformer, dtype='object')- Args:
  - column_names [str]: The names of the column. Should be used to set `self.columns` and `self.output_columns`.
  - min_max_values [tuple or None]: `None` or a tuple containing the (min, max) values for the transformer. Should be used to set `self._min_value` and `self._max_value` and must be provided if `self.enforce_min_max_values` is `True`.
  - null_transformer [a `NullTransformer` instance]: A _fitted_ null transformer instance that can be used to generate null values for the column.
  - dtype [str]: The pandas dtype the reversed data should be converted to. Should be used to set `self._dtype`.

This method should not return anything. It should validate the parameters that are passed in, especially that the null transformer is compatible.

### PR DESCRIPTION
**Pull request by fealho**
CU-86b0pvd4a, Resolve #828 .

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-86b0qcazg SDV-Enterprise - Update DayZ to sample from copulas #600](https://app.clickup.com/t/86b0qcazg)

File: tests/integration/transformers/test_datetime.py [hunk 1]
```diff
@@ -174,6 +175,42 @@ def test_with_enforce_min_max_values_true(self):
...
+        data = pd.DataFrame({
+            'column_name': [
+                pd.to_datetime('2021-01-01'),
+                pd.to_datetime('2022-01-02'),
+                pd.to_datetime('2023-01-03'),
+                pd.to_datetime('2024-01-04'),
+                pd.to_datetime('2025-01-05'),
+                pd.to_datetime('2026-01-06')
+            ]
+        })
```
Comment (gsheni) [inline tests/integration/transformers/test_datetime.py:190]:
```suggestion
        data = pd.DataFrame({
            'column_name': pd.to_datetime([
                '2021-01-01', '2022-01-02', '2023-01-03', 
                '2024-01-04', '2025-01-05', '2026-01-06'
            ])
        })
```

File: tests/integration/transformers/testing.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+data = pd.DataFrame({
+    'column_name': [
+        pd.to_datetime('2021-01-01'),
+        pd.to_datetime('2022-01-02'),
+        pd.to_datetime('2023-01-03'),
+        pd.to_datetime('2024-01-04'),
+        pd.to_datetime('2025-01-05'),
+        pd.to_datetime('2026-01-06')
+    ]
+})
```
Comment (gsheni) [inline tests/integration/transformers/testing.py:15]:
```suggestion
data = pd.DataFrame({
    'column_name': pd.to_datetime([
        '2021-01-01', '2022-01-02', '2023-01-03', 
        '2024-01-04', '2025-01-05', '2026-01-06'
    ])
})
```

File: rdt/transformers/datetime.py [hunk 3]
```diff
@@ -151,6 +152,34 @@ def _fit(self, data):
...
+                A fitted null transformer instance that can be used to generate
+                null values for the column.
+            dtype (str, optional):
+                The dtype to convert the reverse transformed data back to. Defaults to 'object'.
+        """
+        self.reset_randomization()
+        self.columns = [column_name]
+        self.output_columns = [column_name]
+        self._dtype = dtype
+        self.null_transformer = null_transformer
```
Comment (lajohn4747) [inline rdt/transformers/datetime.py:175]: We should handle and test the case where there are null values. `from_column` in NullTransformer adds a new column that should be put into `self.output_columns`.

https://github.com/sdv-dev/RDT/pull/838 for reference

**Review by frances-h — APPROVED** [review]
Looks good, I think you just need to add a unit test for missing the min/max values when `enforce_min_max_values` is `True` for coverage

**Comment by sdv-team** [comment]
Task linked: [CU-86b0pvd4a RDT - Add `set_fitted_parameters` method to `UnixTimestampEncoder` #828](https://app.clickup.com/t/86b0pvd4a)

**Review by lajohn4747 — APPROVED** [review]

**Pull request merged**